parser.go

```diff
@@ -868,6 +868,9 @@ func convertFromSpecificToPrimitive(typeName string) (string, error) {
 }
 
 func (parser *Parser) getTypeSchema(typeName string, file *ast.File, ref bool) (*spec.Schema, error) {
+	if IsInterfaceLike(typeName) {
+		return &spec.Schema{}, nil
+	}
```

Comment:
We need to cover this with unit tests.

Reply:
It should already be covered by `TestGen_ErrorAndInterface` and `TestIsInterfaceLike` but I can recheck.

Reply:
I added a test similar to TestGen_ErrorAndInterface base on testdata in the root package.